bankroll/model/modelconverter.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+from typing import List
+
+from bankroll.model import ConvertableModel, Position, Trade
+
+
+class ModelConverter:
```

Comment:
I'd like to find a better way to solve this than adding the ModelConverter class. Ideas?

Reply:
Why does this need a class at all? Can we use a global function?

Reply:
✅